setup.py

```diff
@@ -0,0 +1,27 @@
+# Minimal setup.py to create a CFFI extension module in abi3 mode if requested
+
+import os
+
+from setuptools import setup
+from setuptools.command.bdist_wheel import bdist_wheel
+
+
+class bdist_wheel_abi3(bdist_wheel):  # noqa: D101
+    def get_tag(self):  # noqa: D102
+        python, abi, plat = super().get_tag()
+
+        if python.startswith("cp"):
+            return "cp310", "abi3", plat
+
+        return python, abi, plat
+
+
+kwargs = dict()
+if os.getenv("MNE_LSL_BUILD_FFI", "0").lower() in ("1", "true"):
+    kwargs.update(
+        cffi_modules=["tools/ffi_build.py:ffibuilder"],
+        cmdclass={"bdist_wheel": bdist_wheel_abi3},
+    )
```

Comment:
Why should we make CFFI build optional here? That would also mean that an editable install would not build everything/work?

Reply:
... see response above, but as `cffi` is just (currently!) a way to force delocation tools to bundle our build library, it's not needed to do anything with `mne-lsl` so long as the library path can be resolved.

> That would also mean that an editable install would not build everything/work?

For editable installs you would need `liblsl.so` on your path somewhere currently. I hadn't thought about this and it's not a great situation. One easy-ish way out of this problem would be to add a `tools/build_lsl.sh` that runs on all three OSes and runs `cibuildwheel` locally to get a suitable DLL in the right place (e.g., `mne_lsl.libs` or `mne_lsl/.dylib` or whatever based on the OS). It's an extra step for devs but it's not too bad I think.

Reply:
Actually, let's say that for now we keep the the `CDLL` and `ctypes` code. Could we not drop `cffi` and all 3 repair-wheel tools `auditwheel` / `delocate-wheel` / `delvewheel` and 'simply' set in `setup.py`:

- Instead of the `before-all` step, run the build of `liblsl` kind of similarly to `antio` here: https://github.com/mscheltienne/antio/blob/9a2bc09499fd0df92f499629d5b8623e91bc24af/setup.py#L34-L85
- Move the build files to `mne_lsl.lsl.lib`
- Include those files in the wheel

That would also give `editable` install out of the box as long as you have a compiler installed?

Reply:
Yes this would probably work!